lib/file/file_windows.go

```diff
@@ -101,3 +112,24 @@ func IsReserved(path string) error {
 	}
 	return nil
 }
+
+// IsLongPathsEnabled checks if the current have the LongPathsEnabled enabled
+func IsLongPathsEnabled() bool {
+	// Read registry
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE, `SYSTEM\CurrentControlSet\Control\FileSystem`, registry.QUERY_VALUE)
+	if err != nil {
+		return false
+	}
+	defer k.Close()
+
+	s, _, err := k.GetIntegerValue("LongPathsEnabled")
+	if err != nil {
+		return false
+	}
+
+	if s == 0x00000001 {
```

Comment:
I'd probably write this as

    return s == 0x00000001

Or maybe

    return s != 0